otterdog/models/repository.py

```diff
@@ -1188,6 +1188,18 @@ async def apply_live_patch(
                 await provider.delete_repo(org_id, unwrap(patch.current_object).name)
 
             case LivePatchType.CHANGE:
+                # if the source path is changed, also ensure that the branch is set when
+                # the branch is not changed. The branch is required by GitHub.
+                if (
+                    patch.changes
+                    and "gh_pages_source_path" in patch.changes
+                    and "gh_pages_source_branch" not in patch.changes
```

Comment:
isn't it an `or` on this line? IIRC, we want to do:

```
if (
	patch.changes
  and (
		"gh_pages_source_path" in patch.changes
		or "gh_pages_source_branch" not in patch.changes
	)
):
```

Reply:
Actually, if I modify the branch I don't want to add it as it will be part of the change.